cluster-autoscaler/FAQ.md

```diff
@@ -62,9 +63,14 @@ Cluster Autoscaler decreases the size of the cluster when some nodes are consist
 
 ### What types of pods can prevent CA from removing a node?
 
-* Kube-system pods that are not run on the node by default.
+* Pods with restrictive PodDisruptionBudget.
+* Kube-system pods that:
+  * are not run on the node by default,
+  * don't have PDB or their PDB is too restrictive (since CA 0.6).
```

Comment:
I'm not sure if it will be clear to the user that for CA 0.6+ both of these must be true at the same time for kube-system pods (instead of either-or)